pkgs/development/ocaml-modules/elina/default.nix

```diff
@@ -12,7 +12,7 @@ stdenv.mkDerivation rec {
 
   propagatedBuildInputs = [ apron camlidl gmp mpfr ];
 
-  strictDeps = true;
+  strictDeps = false;
```

Comment:
Why?

Reply:
For some reason it's not installed properly when it's set to true, I've tried finding the root cause but at some point we just have to un-break it. Having it in this file explicitly helps us grep for it in the future as well.

Reply:
Found the correct fix here as well, it was the same as mlgmpidl apparently...